bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -47,6 +49,15 @@
 				<description>Authentication Token which can be found under "Software Integration" if you connect locally to your
 					sonnen battery. If empty the old deprecated API will be used.</description>
 			</parameter>
+			<parameter name="chargingPower" type="integer" unit="W" min="0" max="10000">
+				<label>sonnen battery maximum charging power</label>
+				<description>Maximum charging power in watt which is supported by sonnen battery type. Be cautious to not enter a
+					higher value then supported by the battery. Details can be found on the datasheet of your battery.
+					Valid input is 0
+					- 10000.</description>
```

Comment:
The range is already contained in the xml / UI and therefore duplicate.
```suggestion
					</description>
```

If needed you could state that 0 is a special case / no limit or what does 0 watt do?

Reply:
ok